config/initializers/menus.rb

```diff
@@ -639,9 +643,9 @@
 
   project_menu_items.each do |key, options|
     menu.push :"settings_#{key}",
-              { controller: "/projects/settings/#{key}", action: "show" },
+              { controller: "/projects/settings/#{key}", action: "show" }.merge(options.slice(:action)),
```

Comment:
```suggestion
              { controller: "/projects/settings/#{key}", action: "show", **options.slice(:action) },
```

Reply:
I find that harder to understand the the merge command.